src/index.js

```diff
@@ -25,7 +25,8 @@ import withTheme from './hoc/withTheme'
 /* Warning if you've imported this file on React Native */
 if (
   process.env.NODE_ENV !== 'production' &&
-  navigator &&
+  typeof navigator === 'object' &&
+  navigator !== null &&
```

Comment:
I think this check should probably on undefined here, right? Probably doesn’t matter.

If you want you can add your name to the existing change log entry

Reply:
I can never pick between just `typeof x === 'undefined'` or the one above. I think it doesn't matter that much in this case, but will change this as the `undefined` check came to your mind first :)